### PR TITLE
Added ostree image compose job

### PR DESCRIPTION
**Pull request by p3ck**
Should run approximately every 24 hours.

### CODE REVIEW

File: tasks/ostree-image-compose [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+# Kill backgrounded jobs on exit
+function clean_up {
+    kill $(jobs -p)
+    sudo rm -f $imgdir/*
+    sudo rm -rf $tmpdir
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+if [ ! -e "Fedora-Everything-netinst-x86_64-26-20170404.n.0.iso" ]; then
```
Comment (johnbieren) [inline tasks/ostree-image-compose]: Is having this dependent on a date okay?

File: tasks/ostree-image-compose [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# Create a tdl file for imagefactory
+#       <install type='url'>
+#           <url>http://download.fedoraproject.org/pub/fedora/linux/releases/25/Everything/x86_64/os/</url>
+#       </install>
+cat <<EOF >./fedora-rawhide.tdl
+<template>
+    <name>rawhide</name>
+    <os>
+        <name>Fedora</name>
+        <version>22</version>
```
Comment (johnbieren) [inline tasks/ostree-image-compose]: version 22? does this matter for anything?

File: tasks/ostree-image-compose [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <arch>x86_64</arch>
+        <install type='iso'>
+            <iso>file://${currentdir}/Fedora-Everything-netinst-x86_64-26-20170404.n.0.iso</iso>
```
Comment (johnbieren) [inline tasks/ostree-image-compose]: date here again, if it matters

File: tasks/ostree-image-compose (hunk 1 above)
Comment (p3ck) [inline tasks/ostree-image-compose]: These have been archived on the artifact server in centos so it should be ok.

File: tasks/ostree-image-compose (hunk 2 above)
Comment (p3ck) [inline tasks/ostree-image-compose]: nobody seems to update this in all the examples I've seen

File: tasks/ostree-image-compose (hunk 3 above)
Comment (p3ck) [inline tasks/ostree-image-compose]: we match what we downloaded above.  That's all that matters.

**Pull request closed without merging**